lighthouse-core/scripts/chromium-web-tests/webtests/http/tests/devtools/lighthouse/lighthouse-emulate-run.js

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
I'm surprised to see these files checked in instead of trying to stay in sync with the devtools tests.

expectations I understand we don't really have a choice since we drift as we go, but do we expect to ever make changes here?

Reply:
ooooooh just saw the `# copy webtests` bit in our roll-to-devtools script.

are we keeping the source of truth tests in our repo now?

Reply:
yup!

The truth can't be CDT frontend, because we want to use the truth tests in our CI, and many of these tests fail when you change the version, default config, etc. Keeping the truth in CDT means we have a circular dependency we can never satisfy.

I wish we could do the same for LR smoke tests. Maybe we could, but it'd require some internal service that runs something on demand of a GitHub action.

Reply:
awesome 👍